test/test_video_feed.py

```diff
@@ -0,0 +1,30 @@
+""" Tests the behavior of the VideoFeed class. """
+
+import pytest
```

Comment:
You are not using anything from this import, please remove

Reply:
fixed in https://github.com/R2D2-2019/vision/pull/4/commits/bc5c6c49bab12a2013078fc8b5abd4564266563f